lib/Doctrine/Persistence/Mapping/AbstractClassMetadataFactory.php

```diff
@@ -400,6 +458,16 @@ public function getReflectionService()
         return $this->reflectionService;
     }
 
+    protected function getCacheKey(string $realClassName): string
+    {
+        return str_replace('\\', '__', $realClassName) . $this->cacheSalt;
+    }
+
+    protected function getClassNameFromCacheKey(string $cacheKey): string
```

Comment:
this is never used, is it worth having it?

Reply:
That may be a leftover of some different API attempts. I'll remove it.